package.json

```diff
@@ -57,7 +57,7 @@
     "@pmmmwh/react-refresh-webpack-plugin": "^0.5.15",
     "@redhat-cloud-services/eslint-config-redhat-cloud-services": "2.0.4",
     "@redhat-cloud-services/frontend-components-config-utilities": "^4.1.5",
-    "@redhat-cloud-services/types": "2.0.0-beta.1",
+    "@redhat-cloud-services/types": "2.0.0-beta.2",
```

Comment:
Confirmed the drawer API was published in master via `1.0.23` and pf6 via `2.0.0-beta.2` of the types package (we need both master and PF6 versions as stage is running PF6 until we merge PF6 back into master):
master: https://github.com/RedHatInsights/frontend-components/commit/34823fc29309314a7f216e72eb99205dd25d2b8c
pf6: https://github.com/RedHatInsights/frontend-components/commit/fadc67b7369be604f169e01d2f8af979d2b1a165